cep-4.md

```diff
@@ -0,0 +1,43 @@
+<table>
+<tr><td> Title </td><td> Deprecate `pip` interoperability and Default `pip` to have `--no-deps` in all environments </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Matthew R. Becker &lt;becker.mr@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sep 15, 2021</td></tr>
+<tr><td> Updated </td><td> Sep 15, 2021</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda/ceps/pull/3 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Installing software from `pip` into `conda` environments creates huge hassels, especially for new users. It 
+can lead to inconsistent or even broken environments with undefined behavior. This CEP proposed to have `conda`'s 
+version of `pip` have the installation of dependencies turned off by default to ease the pain here. Additionally, 
+this CEP proposes to deprecate the current `pip` interoperability feature. A correct implementation of this feature 
+is nearly impossible and partial implementations create as many problems as they solve.
+
+## Specification
+
+1. The `pip` interoperability feature currently in `conda` is deprecated and never to be brought back forevermore.
```

Comment:
This seems a bit overreaching :sweat_smile:. Maybe things will improve enough in the future it would be reasonable to reconsider.

Reply:
Yeah, my understanding is that @beckermr has had plenty of not-so-great experience from helping conda users to fix broken conda environments and adds the bit of snark here to show it. 😬 I can fully appreciate that and think we just need to acknowledge what issues the current feature set brings in real world scenarios, but remove the snark a bit.